nss-c31-tricky-vigor/nutshell
# Rw nutshell events

# Description

I created the add new event button and save button . I also created the input fields for new event name, location, and date. Event listeners were added to both buttons to listen for a click and console log a message showing that they work.

Applies to issue # Events .05

## Type of change






# Testing Instructions

1. `git fetch`
2. `git checkout rw-nutshell-events` 
3. If you already have a reference to this branch, run `git pull origin rw-nutshell-events` to get updates to code.
4. cd into src/lib and run grunt
5. open browser and go to http:localhost:8080
6. verify all forms are appending to the dom and that console log displays messages when buttons are clicked.




# Checklist:
